exercises/numbers/remainderOf/remainderOf.js

```diff
@@ -14,24 +14,31 @@
  *
  * @param {number} num The input number
  * @returns {boolean} True if num is even and false otherwise
+ * @param {remainder} rem The amount leftover 
  */
-function remainderOf(num) {
+function remainderOf(num1, num2) {
   /*
     This is your job. :)
 
     If you're not sure, step out of the code and use pen + paper. Start
     with simple examples and pay attention to the process you carry out.
   */
- return num % 3
+ var result = (num1/num2)
```

Comment:
This is a nice, working solution! But **return num1 % num2** would solve it in one line.